test/controllers/admin_controller_test.rb

```diff
@@ -1,7 +1,65 @@
 require 'test_helper'
 
 class AdminControllerTest < ActionDispatch::IntegrationTest
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
+    login_admin_user
+  end
```

Comment:
We actually have a nice helper for this and a fixture for a pre-existing admin user, so you can do:

```rb
class AdminControllerTest < ActionDispatch::IntegrationTest
  include Devise::Test::IntegrationHelpers

  def setup
    sign_in users(:admin_user)
  end
```

And remove the `login_admin_user` method.